### ISSUE TITLE
Traktflix not showing latest shows watched

### ISSUE THREAD

**Issue by markkhk** [tegon/traktflix#203]
Hello

In last few days using the extension (Edge Version 105.0.1343.53 (Official build) (64-bit)

I keep getting message:
"There's no more items in your history. Refresh the page to go to the beginning."

Have logged out of service, refreshed, restarted, etc.  

Are others encountering this issue?

**Comment by mrkfromsd** [comment]
I've been getting the same message on both the Firefox and Chrome Extensions.

**Comment by boris377** [comment]
Same here.  It connects to the main page (shows it is connected and see netflix) and then once i start anything it just sits there.

**Comment by jwchia1** [comment]
yup, been having the same issues on the latest version of chrome
i think there's some issue with contacting netflix api (404), saw it in the console

**Comment by christophermoura** [comment]
> 

I'm having the same issue. I've tested it in Chrome and Firefox.

I've checked the console output on DevTools and saw the same error as [mentioned above](https://github.com/tegon/traktflix/issues/203#issuecomment-1267231706) by @jwchia1: `Failed to load resource: the server responded with a status of 404 ()`.

```
404 '<!doctype html><html lang="en"><head><title>HTTP Status 404 – Not Found</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 404 – Not Found</h1></body></html>'
```

**Comment by slunat** [comment]
I'm seeing the same issue, can't see any of my Netflix watch history in the plugin so can't sync it back to Trakt. Really frustrating. I get a popup that says "Failed to retrieve history :( we will work on a solution soon"

**Comment by joaociocca** [comment]
One more here. Edge 106.0.1370.37 64 bit, with Traktflix version 2.1.1

**Comment by christophermoura** [comment]
I found that this project has been discontinued in favor of another project, created by the same developer/maintainer (@tegon): https://github.com/trakt-tools/universal-trakt-scrobbler

> **_"This extension is based on traktflix, the original Netflix sync developed by tegon, which was discontinued in favor of Universal Trakt Sync."_**
